### PR TITLE
docs: add import and deploy OVA guide DOC-829

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR adds the import and deploy OVA guide. 

## Changed Pages

💻 [Import and Deploy OVAs to Palette VMO](https://deploy-preview-3909--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/advanced-topics/deploy-import-ova/)

## Jira Tickets

🎫 [DOC-829](https://spectrocloud.atlassian.net/browse/DOC-829)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
To all 4.x branches

[DOC-829]: https://spectrocloud.atlassian.net/browse/DOC-829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+   then need to supply the corresponding private key when connecting to the VM.
+
+   ```shell
+   virtctl scp public-key.pub root@vmo-jh.default:/root
+
+   virtctl ssh root@vmo-jh.default --local-ssh=true
+
+   mkdir -p /root/.ssh && cat /root/public-key.pub >> /root/.ssh/authorized_keys
+   ```
+
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:149]: Please add this warning:

> If SSH-ing from a Windows machine, omit the `--local-ssh=true` flag

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+6. Copy your cluster's kubeconfig file to the VM. Replace the placeholders with local path to the kubeconfig file and
+   the path to your private key. The kubeconfig file is required for subsequent operations.
+
+   ```shell
+   virtctl scp <path-to-downloaded-kubeconfig-file> -i <path-to-private-key> root@vmo-jh.default:/root
+   ```
+
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:156]: Please insert a step here explaining that you must download the latest palette CLI release binary to the jump host.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+    | **Kubeconfig Path**                         | The path to the kubeconfig file you have uploaded to your VM.                                                                                                                                     |                                   |
+    | **DataVolume Namespace**                    | The namespace to create your `DataVolume`, if you selected this option previously.                                                                                                                |                                   |
+    | **DataVolume Name**                         | The name of your `DataVolume`.                                                                                                                                                                    |                                   |
+    | **Overhead Percentage for DataVolume Size** | Set an overhead percentage for your `DataVolume` compared to the OVA specification. This parameter is optional and can be skipped with the value `-1`.                                            |                                   |
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:202]: Suggestion:

> If skipped, the filesystem overhead percentage will be inferred from the CDI Custom Resource in your VMO cluster. Refer to the [CDI configuration options](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/cdi-config.md#options) for additional info.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    | **Access Mode for the PVC**                 | Set the access mode for your `DataVolume`. Ensure that your configured CSI supports your selection.                                                                                               | `ReadWriteMany` / `ReadWriteOnce` |
+    | **Create a PVC with VolumeMode=Block**      | Indicate whether to set `Block` volume mode on the `DataVolume`.                                                                                                                                  | `y` / `N`                         |
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:204]: If yes is chosen for this prompt, the containerd config patch documented elsewhere must be in place!

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+
+    The Palette CLI prompts you for information regarding the VM you want to deploy.
+
+    | **Parameter**               | **Description**                                   | **Values** |
+    | --------------------------- | ------------------------------------------------- | ---------- |
+    | **VM Namespace**            | The namespace to create your VM into.             |            |
+    | **VM Name**                 | The namespace of your VM.                         |            |
+    | **Start VM Automatically?** | Indicate whether to start your VM after creation. | `y`/ `N`   |
+
+13. Depending on the configuration of your OVA, you may be prompted to provide value. The deployment of your VM should
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:227]: Suggestion: "to provide value" -> to provide additional values for the OVF template variables

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+    | **Create a PVC with VolumeMode=Block**      | Indicate whether to set `Block` volume mode on the `DataVolume`.                                                                                                                                  | `y` / `N`                         |
+    | **StorageClass**                            | The storage class on the destination that will be used to create the VM volume.                                                                                                                   |                                   |
+    | **CDI upload proxy URL**                    | Optionally, provide a URL to upload the CDI custom resource.                                                                                                                                      |                                   |
+
+11. The import may take a few minutes to complete. The Palette CLI outputs the path for your OVA configuration file.
+    Make a note of it.
+
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:210]: We may want to include the following warning:
> virt-v2v may emit a variety of warnings, including many repeated lines containing the word (BADINDEX). All of these warnings and messages can safely be ignored. If a fatal error is encountered the conversion will exit completely.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+12. Your imported OVA is now ready to be deployed to a new VM inside your VMO cluster. Execute the following command to
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:211]: We may want to include the following warning:
> The DataVolume upload will fail if you are in a proxied environment and have not properly configured the CDI custom resource. Refer to the [CDI configuration options](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/cdi-config.md#options) as required.
>
> If you have configured NGINX ingress + MetalLB and you can access the NGINX load balancer IP, but not the CDI upload proxy IP, add an entry to `/etc/hosts` that maps the CDI upload proxy host name, e.g. `cdi-uploadproxy.mycompany.io`, to the NGINX load balancer’s public IP.

Comment (TylerGillson) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:211]: And we should definitely add this info box:

> If the DataVolume upload fails for any reason, re-run the upload after debugging any CDI issues using: `palette vmo import-ova -s -f /path-to-ova-config.yaml --skip-convert`

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+
+    | **Parameter**               | **Description**                                   | **Values** |
+    | --------------------------- | ------------------------------------------------- | ---------- |
+    | **VM Namespace**            | The namespace to create your VM into.             |            |
+    | **VM Name**                 | The namespace of your VM.                         |            |
+    | **Start VM Automatically?** | Indicate whether to start your VM after creation. | `y`/ `N`   |
+
+13. Depending on the configuration of your OVA, you may be prompted to provide value. The deployment of your VM should
+    complete within a few minutes.
+
```
Comment (TylerGillson) [inline docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:229]: We may want to call out the fact that the proposed VM spec will be opened in a text editor and that at a minimum, you must delete the first two lines of the file.

Info:
> Minor edits to the auto-generated VM spec may be required; e.g., you may need to specify Multus VLAN configuration for each network interface, as the CLI does not yet support eliciting this information during the import process.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md (hunk 2 above)
Comment (TylerGillson) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:156]: At first I was, then I decided that I don't want to have to rebuild the jump host image each time we release the CLI.

Comment (addetz) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:212]: Added this step now.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md (hunk 4 above)
Comment (addetz) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md]: I have added the same containerd config patch from the migration here.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md (hunk 7 above)
Comment (addetz) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md]: Added thank you so much!

**Review by karl-cardenas-coding — APPROVED** [review]
Awesome addition

File: docs/docs-content/automation/palette-cli/commands/vmo.md
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/automation/palette-cli/commands/vmo.md:1]: Thanks for catching this! Will add.

File: docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md
[the remark this answers is not in the document]
Comment (addetz) [reply docs/docs-content/vm-management/create-manage-vm/advanced-topics/deploy-import-ova.md:1]: I thought i got them all. Fixing.

**Pull request merged**